rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -1554,6 +1555,25 @@ String loginOrRenewSession(@HeaderParam("sessionid") String sessionId, @FormPara
 	String loginOrRenewSession(@HeaderParam("sessionid") String sessionId, @MultipartForm LoginForm multipart)
 			throws KeyException, SchedulerRestException, LoginException, NotConnectedRestException;
 
+    /**
+     * Get the login string associated to the {@code sessionId} if it exists
+     * 
+     * In case that the given sessionId doesn't have an associated login (session id expired, or invalid), 
+     * this endpoint will throw a {@link javax.ws.rs.NotFoundException}
+     * 
+     * @param sessionId with which the endpoint is going to look for the login value
+     * @return the associated login value
+     * @throws SchedulerRestException
+     * @throws LoginException
+     * @throws NotConnectedRestException
+     * @throws NotFoundException if the given session id doesn't have an associated login value found
+     */
+    @GET
+    @Path("logins/sessionid/{sessionId}")
+    @Produces("application/json")
+    String getLoginFromSessionId(@PathParam("sessionId") String sessionId)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: org.ow2.proactive_grid_cloud_portal.scheduler.exception.SchedulerRestException, javax.security.auth.login.LoginException, org.ow2.proactive_grid_cloud_portal.scheduler.exception.NotConnectedRestException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1160)